src/CollAction/ValidationAttributes/FileTypeAttribute.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.IO;
+using System.Linq;
+
+namespace CollAction.ValidationAttributes
+{
+    public class FileTypeAttribute : ValidationAttribute
```

Comment:
My bad, shouldn't search for FileTypeAttribute, but FileType.

One small thing, if you make the constructor to be this:

 public FileTypeAttribute(params[] string types)
 {
     _types = types;
 }

Then you could use the attribute like this:
[FileType("jpg", "png")]

https://msdn.microsoft.com/nl-nl/library/w5zay9db.aspx